components/04-views/catalog/_catalog.scss

```diff
@@ -1,37 +1,55 @@
+//product-grid
+$product-grid-padding             : 10px !default;
+$product-grid-margin              : 15px !default;
+$product-grid-breakpoint-medium   : $screen-m !default;
+$product-grid-breakpoint-large    : $screen-l !default;
+
+//product-list
+$product-list-padding             : 10px !default;
+$product-list-margin              : 10px !default;
+$product-list-border-color        : $gray-light !default;
```

Comment:
I don't want to reapeat myself N more time, so just remove all separate variables for border-colors

Reply:
ALL DELETED